spec/models/decision_spec.rb

```diff
@@ -26,6 +26,15 @@
     end
 
```

Comment:
It would be good to include a test for the model being invalid. E.g. when the user is an assessor, it's refused, and there's an empty string as the comment.